packages/cube/README.md

```diff
@@ -6,80 +6,116 @@ The `manifest.ttl` file contains a full list of all operations included in this
-```
+For big cubes, it is advisable to use the other fetch commands to separately get the cube parts (for example `fetch-metadata` and `fetch-observations`).
 
-## Cube validation
+### fetch constraint
 
-The following pipelines retrieve and validate cube observations and metadata.
+Pipeline `fetch-constraint` queries a given SPARQL endpoint to retrieve 
+a [concise bounded description](https://docs.stardog.com/query-stardog/#describe-queries) of the constraint of a given cube.
+
+```bash
+barnard59 cube fetch-constraint \
+  --cube https://agriculture.ld.admin.ch/agroscope/PRIFm8t15/2 \
+  --endpoint https://int.lindas.admin.ch/query
+```
 
 ### fetch metadata
 
 Pipeline `fetch-metadata` queries a given SPARQL endpoint to retrieve 
-a [concise bounded description](https://docs.stardog.com/query-stardog/#describe-queries) of a given cube and its constraint (excluding the observations).
+a concise bounded description of a given cube and its constraint (excluding the observations).
 
 ```bash
 barnard59 cube fetch-metadata \
   --cube https://agriculture.ld.admin.ch/agroscope/PRIFm8t15/2 \
-  --endpoint https://int.lindas.admin.ch/query
+  --endpoint https://int.lindas.admin.ch/query \
+> metadata.ttl
 ```
 
+### fetch observations
+
+Pipeline `fetch-observations` queries a given SPARQL endpoint to retrieve the observations of a given cube.
 
-This pipeline is useful mainly for cubes published with [cube creator](https://github.com/zazuko/cube-creator) (if the cube definition is manually crafted, likely it's already available as a local file).
+```bash
+barnard59 cube fetch-observations \
+    --cube https://agriculture.ld.admin.ch/agroscope/PRIFm8t15/2 \
+    --endpoint https://int.lindas.admin.ch/query \
+ > observations.ttl
+```
+
+The output of all the fetch pipelines is written to `stdout` and can be redirected to a local file as in the last examples.
 
 
 ### check metadata
 
-Pipeline `check-metadata` validates the input metadata against the shapes provided with the `profile` variable (the default profile is https://cube.link/latest/shape/standalone-constraint-constraint).
+Pipeline `check-metadata` validates the input metadata against the shapes provided with the `--profile` option (the default profile is https://cube.link/latest/shape/standalone-constraint-constraint).
 
-The pipeline reads the metadata from `stdin`, allowing input from a local file (as in the following example) as well as from the output of the `fetch-metadata` pipeline (in most cases it's useful to have the metadata in a local file because it's needed also for the `check-observations` pipeline).
+The pipeline reads the metadata from `stdin`, allowing input from a local file (as in the following example) as well as directly from the output of the `fetch-metadata` pipeline (in most cases it's useful to have the metadata in a local file because it's needed also for the `check-observations` pipeline).
```

Comment:
```suggestion
The pipeline reads the metadata from `stdin`, allowing input from a local file (as in the following example), as well as directly from the output of the `fetch-metadata` pipeline. In most cases it's useful to have the metadata in a local file because it's needed also for the `check-observations` pipeline.
```